<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>UsersWindow</class>
    <widget class="QMainWindow" name="UsersWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1064</width>
                <height>235</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>لیست ساکنین شهرک</string>
        </property>
        <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QLabel" name="label_3">
                <property name="geometry">
                    <rect>
                        <x>440</x>
                        <y>130</y>
                        <width>91</width>
                        <height>29</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                    </font>
                </property>
                <property name="text">
                    <string>وضعیت تردد</string>
                </property>
                <property name="alignment">
                    <set>Qt::AlignCenter</set>
                </property>
            </widget>
            <widget class="QLabel" name="plateTextView">
                <property name="enabled">
                    <bool>true</bool>
                </property>
                <property name="geometry">
                    <rect>
                        <x>540</x>
                        <y>160</y>
                        <width>300</width>
                        <height>62</height>
                    </rect>
                </property>
                <property name="sizePolicy">
                    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                    </sizepolicy>
                </property>
                <property name="maximumSize">
                    <size>
                        <width>300</width>
                        <height>62</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                        <pointsize>29</pointsize>
                        <kerning>true</kerning>
                    </font>
                </property>
                <property name="layoutDirection">
                    <enum>Qt::LeftToRight</enum>
                </property>
                <property name="autoFillBackground">
                    <bool>false</bool>
                </property>
                <property name="styleSheet">
                    <string notr="true"/>
                </property>
                <property name="frameShape">
                    <enum>QFrame::Box</enum>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="textFormat">
                    <enum>Qt::PlainText</enum>
                </property>
                <property name="scaledContents">
                    <bool>false</bool>
                </property>
                <property name="alignment">
                    <set>Qt::AlignCenter</set>
                </property>
            </widget>
            <widget class="QLabel" name="label_6">
                <property name="geometry">
                    <rect>
                        <x>890</x>
                        <y>130</y>
                        <width>141</width>
                        <height>29</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                    </font>
                </property>
                <property name="text">
                    <string>نوع خودرو</string>
                </property>
                <property name="alignment">
                    <set>Qt::AlignCenter</set>
                </property>
            </widget>
            <widget class="QLabel" name="label_5">
                <property name="geometry">
                    <rect>
                        <x>610</x>
                        <y>130</y>
                        <width>181</width>
                        <height>21</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                    </font>
                </property>
                <property name="text">
                    <string>پلاک خودرو</string>
                </property>
                <property name="alignment">
                    <set>Qt::AlignCenter</set>
                </property>
            </widget>
            <widget class="QWidget" name="layoutWidget">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>10</y>
                        <width>1041</width>
                        <height>31</height>
                    </rect>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="label_4">
                            <property name="font">
                                <font>
                                    <family>B Yekan</family>
                                </font>
                            </property>
                            <property name="text">
                                <string>نام</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLabel" name="label_10">
                            <property name="font">
                                <font>
                                    <family>B Yekan</family>
                                </font>
                            </property>
                            <property name="text">
                                <string>نام خانوادگی</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="2">
                        <widget class="QLabel" name="label_2">
                            <property name="font">
                                <font>
                                    <family>B Yekan</family>
                                </font>
                            </property>
                            <property name="text">
                                <string>ساختمان</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="4">
                        <widget class="QLabel" name="label_7">
                            <property name="font">
                                <font>
                                    <family>B Yekan</family>
                                </font>
                            </property>
                            <property name="text">
                                <string>پلاک</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="3">
                        <widget class="QLabel" name="label_8">
                            <property name="font">
                                <font>
                                    <family>B Yekan</family>
                                </font>
                            </property>
                            <property name="text">
                                <string>بلوک</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                </layout>
            </widget>
            <widget class="QPushButton" name="addResidentButton">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>170</y>
                        <width>131</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                    </font>
                </property>
                <property name="text">
                    <string>ثبت ساکن</string>
                </property>
            </widget>
            <widget class="plateQLineEdit" name="fNameTextBox">
                <property name="geometry">
                    <rect>
                        <x>850</x>
                        <y>50</y>
                        <width>200</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="baseSize">
                    <size>
                        <width>200</width>
                        <height>50</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                    </font>
                </property>
                <property name="cursor">
                    <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);
                        border-color: rgb(0, 0, 0);
                        border-width: 1px;
                        border-style: solid;
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="maxLength">
                    <number>50</number>
                </property>
                <property name="frame">
                    <bool>false</bool>
                </property>
            </widget>
            <widget class="plateQLineEdit" name="lNameTextBox">
                <property name="geometry">
                    <rect>
                        <x>640</x>
                        <y>50</y>
                        <width>200</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="baseSize">
                    <size>
                        <width>200</width>
                        <height>50</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                    </font>
                </property>
                <property name="cursor">
                    <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);
                        border-color: rgb(0, 0, 0);
                        border-width: 1px;
                        border-style: solid;
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="maxLength">
                    <number>50</number>
                </property>
                <property name="frame">
                    <bool>false</bool>
                </property>
            </widget>
            <widget class="plateQLineEdit" name="buildingTextBox">
                <property name="geometry">
                    <rect>
                        <x>430</x>
                        <y>50</y>
                        <width>200</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="baseSize">
                    <size>
                        <width>200</width>
                        <height>50</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                    </font>
                </property>
                <property name="cursor">
                    <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);
                        border-color: rgb(0, 0, 0);
                        border-width: 1px;
                        border-style: solid;
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="maxLength">
                    <number>3</number>
                </property>
                <property name="frame">
                    <bool>false</bool>
                </property>
            </widget>
            <widget class="plateQLineEdit" name="blockTextBox">
                <property name="geometry">
                    <rect>
                        <x>220</x>
                        <y>50</y>
                        <width>200</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="baseSize">
                    <size>
                        <width>200</width>
                        <height>50</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                    </font>
                </property>
                <property name="cursor">
                    <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);
                        border-color: rgb(0, 0, 0);
                        border-width: 1px;
                        border-style: solid;
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="maxLength">
                    <number>3</number>
                </property>
                <property name="frame">
                    <bool>false</bool>
                </property>
            </widget>
            <widget class="plateQLineEdit" name="numTextBox">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>50</y>
                        <width>200</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="baseSize">
                    <size>
                        <width>200</width>
                        <height>50</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                    </font>
                </property>
                <property name="cursor">
                    <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);
                        border-color: rgb(0, 0, 0);
                        border-width: 1px;
                        border-style: solid;
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="maxLength">
                    <number>3</number>
                </property>
                <property name="frame">
                    <bool>false</bool>
                </property>
            </widget>
            <widget class="plateQLineEdit" name="carModelTextBox">
                <property name="geometry">
                    <rect>
                        <x>850</x>
                        <y>170</y>
                        <width>200</width>
                        <height>50</height>
                    </rect>
                </property>
                <property name="baseSize">
                    <size>
                        <width>200</width>
                        <height>50</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                    </font>
                </property>
                <property name="cursor">
                    <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);
                        border-color: rgb(0, 0, 0);
                        border-width: 1px;
                        border-style: solid;
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="maxLength">
                    <number>50</number>
                </property>
                <property name="frame">
                    <bool>false</bool>
                </property>
            </widget>
            <widget class="plateQLineEdit" name="plateTextNum_4">
                <property name="geometry">
                    <rect>
                        <x>780</x>
                        <y>180</y>
                        <width>41</width>
                        <height>31</height>
                    </rect>
                </property>
                <property name="baseSize">
                    <size>
                        <width>200</width>
                        <height>50</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                        <pointsize>12</pointsize>
                    </font>
                </property>
                <property name="cursor">
                    <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);
                        border-color: rgb(0, 0, 0);
                        border-width: 1px;
                        border-style: solid;

                        letter-spacing: 3px;
                        text-align: center;
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="maxLength">
                    <number>2</number>
                </property>
                <property name="frame">
                    <bool>false</bool>
                </property>
            </widget>
            <widget class="plateQLineEdit" name="plateTextNum_3">
                <property name="geometry">
                    <rect>
                        <x>700</x>
                        <y>170</y>
                        <width>61</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="baseSize">
                    <size>
                        <width>200</width>
                        <height>50</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                        <pointsize>14</pointsize>
                    </font>
                </property>
                <property name="cursor">
                    <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);
                        border-color: rgb(0, 0, 0);
                        border-width: 1px;
                        border-style: solid;

                        letter-spacing: 3px;
                        text-align: center;
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="maxLength">
                    <number>3</number>
                </property>
                <property name="frame">
                    <bool>false</bool>
                </property>
            </widget>
            <widget class="plateQLineEdit" name="plateTextNum_1">
                <property name="geometry">
                    <rect>
                        <x>580</x>
                        <y>170</y>
                        <width>51</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="baseSize">
                    <size>
                        <width>200</width>
                        <height>50</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                        <pointsize>14</pointsize>
                    </font>
                </property>
                <property name="cursor">
                    <cursorShape>ArrowCursor</cursorShape>
                </property>
                <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);
                        border-color: rgb(0, 0, 0);
                        border-width: 1px;
                        border-style: solid;

                        letter-spacing: 3px;
                        text-align: center;
                    </string>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="maxLength">
                    <number>2</number>
                </property>
                <property name="frame">
                    <bool>false</bool>
                </property>
            </widget>
            <widget class="QComboBox" name="plateAlphabetComboBox">
                <property name="geometry">
                    <rect>
                        <x>640</x>
                        <y>170</y>
                        <width>51</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="baseSize">
                    <size>
                        <width>200</width>
                        <height>50</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                        <pointsize>11</pointsize>
                    </font>
                </property>
                <property name="styleSheet">
                    <string notr="true"/>
                </property>
            </widget>
            <widget class="QComboBox" name="statusComboBox">
                <property name="geometry">
                    <rect>
                        <x>440</x>
                        <y>170</y>
                        <width>91</width>
                        <height>41</height>
                    </rect>
                </property>
                <property name="baseSize">
                    <size>
                        <width>200</width>
                        <height>50</height>
                    </size>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                        <pointsize>11</pointsize>
                    </font>
                </property>
                <property name="styleSheet">
                    <string notr="true"/>
                </property>
            </widget>
            <widget class="QLabel" name="statusLabel">
                <property name="geometry">
                    <rect>
                        <x>150</x>
                        <y>180</y>
                        <width>281</width>
                        <height>29</height>
                    </rect>
                </property>
                <property name="font">
                    <font>
                        <family>B Yekan</family>
                    </font>
                </property>
                <property name="styleSheet">
                    <string notr="true"/>
                </property>
                <property name="text">
                    <string/>
                </property>
                <property name="alignment">
                    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
            </widget>
            <zorder>layoutWidget</zorder>
            <zorder>label_3</zorder>
            <zorder>plateTextView</zorder>
            <zorder>label_6</zorder>
            <zorder>label_5</zorder>
            <zorder>addResidentButton</zorder>
            <zorder>fNameTextBox</zorder>
            <zorder>lNameTextBox</zorder>
            <zorder>buildingTextBox</zorder>
            <zorder>blockTextBox</zorder>
            <zorder>numTextBox</zorder>
            <zorder>carModelTextBox</zorder>
            <zorder>plateTextNum_4</zorder>
            <zorder>plateTextNum_3</zorder>
            <zorder>plateTextNum_1</zorder>
            <zorder>plateAlphabetComboBox</zorder>
            <zorder>statusComboBox</zorder>
            <zorder>statusLabel</zorder>
        </widget>
    </widget>
    <customwidgets>
        <customwidget>
            <class>plateQLineEdit</class>
            <extends>QLineEdit</extends>
            <header>gui.plateQLineEdit</header>
        </customwidget>
    </customwidgets>
    <tabstops>
        <tabstop>addResidentButton</tabstop>
    </tabstops>
    <resources/>
    <connections/>
</ui>
